.gitignore

```diff
@@ -184,3 +184,7 @@ event.json
 test_websocket.py
 test_raw.py
 rawdata.json
+
```

Comment:
Please do this in a separate PR as its not related to the new feature being added

Reply:
Sorry about that.  Still getting my environment organized, and meant to push this to a different branch.  I will fix that.

Reply:
Its no problem to add these if you do want to do a separate PR for them.